src/sass/components/_product-box.scss

```diff
@@ -3,11 +3,35 @@
   border: 1px solid #e1e1e1;
   margin-bottom: 2rem;
 
+  .photo1 {
```

Comment:
jeśli obrazki nazywają się zgodnie z numeracją to może warto byłoby pomyśleć o jakiejś pętli? Kod byłby wtedy krótszy i bardziej uniwersalny

Reply:
dodałam pętlę, chodziło o taką, czy coś więcej?

Reply:
Chodzi o pętlę w Sassie. 
Odsyłam do dokumentacji -> http://thesassway.com/intermediate/if-for-each-while

Reply:
Dodałam pętle, jednak nie rozdziela ona zdjęć tak jak powinna. Czy mógłby ktoś na to zerknąć i udzielić mi uwag?